data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/AggregateProcessorConfig.java

```diff
@@ -39,7 +39,7 @@ public class AggregateProcessorConfig {
     private PluginModel aggregateAction;
 
     @JsonPropertyDescription("The amount of time that a group should exist before it is concluded automatically. Supports ISO_8601 notation strings (\"PT20.345S\", \"PT15M\", etc.) as well as simple notation for seconds (\"60s\") and milliseconds (\"1500ms\"). Default value is 180s.")
-    @JsonProperty("group_duration")
+    @JsonProperty(value = "group_duration", defaultValue = DEFAULT_GROUP_DURATION_SECONDS + "s")
```

Comment:
We don't need string conversion of `DEFAULT_GROUP_DURATION_SECONDS` here?

Reply:
The default value itself is `180s` because it is a duration value.